### PR TITLE
Winning conditions

### PR DESCRIPTION
**Pull request by Simonolsson98**
changed rb back to transform in Move method of FlockAgent, removed Debug.Log on ExitGame.cs, removed script on camera that gave errors

### CODE REVIEW

**Comment by maxarfvidssonnilsson** [comment]
The game is not reset when going back to meny and playing again. So there are units left on the filed from the last game when playing a new one.

**Comment by maxarfvidssonnilsson** [comment]
The map is also not reset so there now spawn a new map on top of the old one.

File: KA-Flocking/Assets/Scripts/Flock.cs [hunk 1]
```diff
@@ -99,5 +99,11 @@ public void RemoveUnit(FlockAgent agent)
     private void RemoveUnitsFromFlock()
     {
         _agents.RemoveAll(deadUnits.Contains);
+        foreach (FlockAgent deadUnit in deadUnits){ //for testing purposes, until we know what to do with dead units
+            deadUnit.gameObject.SetActive(false); //make dead troops invisible
+
```
Comment (Hoidi) [inline KA-Flocking/Assets/Scripts/Flock.cs]: empty line

File: KA-Flocking/Assets/Scripts/ScorePanel.cs [hunk 2]
```diff
@@ -17,5 +19,12 @@ public class ScorePanel : MonoBehaviour
     void Update()
     {
         infantryNoText.text = flock.agents.Count.ToString();
+        if(flock.agents.Count == 0)
+        {
+
```
Comment (Hoidi) [inline KA-Flocking/Assets/Scripts/ScorePanel.cs]: empty line again

**Pull request merged**